lib/cypress/app_config.rb

```diff
@@ -40,5 +40,12 @@ def self.[]=(key, val)
       # Allow setting app config during app init, before Rails.cache exists
       clear
     end
+
+    def self.load_yml
+      app_config = YAML.load(ERB.new(File.read("#{Rails.root}/config/cypress.yml")).result)
+      names_config = YAML.load(ERB.new(File.read("#{Rails.root}/config/names.yml")).result)
+      app_config['randomization']['names'] = names_config['names']
```

Comment:
I'm not speaking for Rob but I don't think it's for neatness as much as doing it with a global constant avoids some unnecessary loading of the file (since the values never change at run time). I originally suggested you do it this way because like you said it's where the names were loaded previously, but I have no issue with Rob's method.